File: samples/client/3_1_0_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/ArrayTypeMatchesArrays.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.ListJsonSchema;
import org.openapijsonschematools.client.schemas.validation.FrozenList;

public class ArrayTypeMatchesArrays {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class ArrayTypeMatchesArrays1 extends ListJsonSchema.ListJsonSchema1 {
        private static @Nullable ArrayTypeMatchesArrays1 instance = null;
        public static ArrayTypeMatchesArrays1 getInstance() {
            if (instance == null) {
                instance = new ArrayTypeMatchesArrays1();
            }
            return instance;
        }
    }

}

File: samples/client/3_1_0_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/ArrayTypeMatchesArrays.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.ListJsonSchema
import org.openapijsonschematools.client.schemas.validation.FrozenList

class ArrayTypeMatchesArrays: ListJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class ArrayTypeMatchesArrays1 : ListJsonSchema.ListJsonSchema1() {
        companion object {
            @Volatile
            private var instance: ArrayTypeMatchesArrays1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: ArrayTypeMatchesArrays1().also { instance = it }
                }
        }
    }

}
